lib/view/formulier/InvoerVelden.class.php

```diff
@@ -838,7 +838,8 @@ public function getHtml() {
 		$html = '<input name="' . $this->name . '_show" value="' . $show_value . '" origvalue="' . $this->show_value . '"' . $this->getInputAttribute(array('type', 'id', 'class', 'disabled', 'readonly', 'maxlength', 'placeholder', 'autocomplete')) . ' />';
 
 		// actual values
-		$class = $this->model->orm;
+        $model = $this->model;
```

Comment:
Hier zit er nog een $class = $this->model->orm;
https://github.com/csrdelft/csrdelft.nl/blob/779d61bc03c4eedc673e3eac5ffc7753f6c6fe25/lib/view/formulier/InvoerVelden.class.php#L820